src/common/types/feed.js

```diff
@@ -39,8 +39,14 @@ type Comment = {
     is_reply_comment?: boolean,
     message?: string,
     modified_at: string,
+    parent?: {
+        id: string,
+        type: 'comment' | 'annotation',
+    },
     permissions: BoxCommentPermission,
+    replies?: Array<Comment>,
     tagged_message: string,
```

Comment:
Will these unused fields be included in the new schema? Or should we be using a new "Comment" type for these flows?

Reply:
I don't think these fields are included in our new backend schema for these objects, though. We should have accurate types, even if it's a bit more work.

Reply:
Did we come to a resolution on this question?